src/components/pages/test-pages/rating-page/RatingTestPage.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const wrapper: SxProps<Theme> = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: 'white',
```

Comment:
Delete this `backgroundColor`. We don`t use white color for production and dev environments.